pkg/karmadactl/unjoin.go

```diff
@@ -40,23 +39,19 @@ func NewCmdUnjoin(cmdOut io.Writer, karmadaConfig KarmadaConfig, cmdStr string)
 	opts := CommandUnjoinOption{}
 
 	cmd := &cobra.Command{
-		Use:     "unjoin CLUSTER_NAME --cluster-kubeconfig=<KUBECONFIG>",
-		Short:   unjoinShort,
-		Long:    unjoinLong,
-		Example: getUnjoinExample(cmdStr),
-		Run: func(cmd *cobra.Command, args []string) {
-			err := opts.Complete(args)
-			if err != nil {
-				klog.Fatalf("Error: %v", err)
+		Use:          "unjoin CLUSTER_NAME --cluster-kubeconfig=<KUBECONFIG>",
+		Short:        unjoinShort,
+		Long:         unjoinLong,
+		Example:      getUnjoinExample(cmdStr),
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Complete(args); err != nil {
+				return err
 			}
-			if errs := opts.Validate(); len(errs) != 0 {
-				klog.Fatalf("Error: %v", utilerrors.NewAggregate(errs).Error())
-			}
```

Comment:
I don't think this `Validate` should be removed or merged into `Complete`.

Reply:
Thank you for review it carefully!
It is an omit when I merging commit. I will fix as soon as possible. By the way, which `Validate()` do we adopt，`error` or `[]error`, I will update them according to your optioning.

Reply:
> By the way, which Validate() do we adopt，error or []error, I will update them according to your optioning.

to `error`. I tried to aggregate the `[]error`, but not work well.

Reply:
Ok, I will do it, thanks.